drop-in/src/main/java/com/adyen/checkout/dropin/ui/DropInActivity.kt

```diff
@@ -353,6 +354,7 @@ class DropInActivity : AppCompatActivity(), DropInBottomSheetDialogFragment.Prot
         hideAllScreens()
         val dialogFragment = when (storedPaymentMethod.type) {
             PaymentMethodTypes.SCHEME -> CardComponentDialogFragment
+            PaymentMethodTypes.BACS -> BacsDirectDebitDialogFragment
```

Comment:
I think this should be removed, we do not support stored BACS payment method now right?